crates/parser/src/config/snapshots/parser__config__test__config_schema_is_generated.snap

```diff
@@ -15,86 +15,89 @@ expression: schema
       "oneOf": [
         {
           "title": "GZip",
+          "default": "gzip",
           "const": "gzip"
         },
         {
           "title": "Zip Archive",
+          "default": "zip",
           "const": "zip"
         },
         {
           "title": "Zstandard",
+          "default": "zstd",
           "const": "zstd"
         },
         {
           "title": "None",
+          "default": "none",
           "const": "none"
         },
         {
           "title": "Auto",
+          "default": null,
           "const": null
         }
       ]
     },
     "format": {
       "description": "Determines how to parse the contents. The default, 'Auto', will try to determine the format automatically based on the file extension or MIME type, if available.",
       "default": {
-        "auto": {}
+        "type": "auto"
       },
       "oneOf": [
         {
           "title": "Auto",
           "description": "Attempt to determine the format automatically, based on the file extension or associated content-type.",
           "type": "object",
           "required": [
-            "auto"
+            "type"
           ],
           "properties": {
-            "auto": {
-              "default": {},
-              "type": "object"
+            "type": {
+              "default": "auto",
+              "const": "auto"
             }
-          },
-          "additionalProperties": false
```

Comment:
Is it okay that we don't have this anymore?

Reply:
Yeah, I don't think disallowing additional properties really gets us much benefit, and it makes it more complicated to upgrade connectors since you can never remove properties from the schema without breaking existing configs.